### PR TITLE
Delete unused packages from 7.0 branch

### PR DESCRIPTION
**Pull request by xugao**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ yarn change`

#### Description of changes
Deleted packages which is unrelated to `@fluentui/react` version 7:
`@fluentui/codemods`
`@fluentui/common-styles`
`@fluentui/ie11-polyfills`
`@fluentui/react-avatar`
`@fluentui/react-button`
`@fluentui/react-checkbox`
`@fluentui/react-compose`
`@fluentui/react-icons`
`@fluentui/react-image`
`@fluentui/react-link`
`@fluentui/react-next`
`@fluentui/react-slider`
`@fluentui/react-stylesheets`
`@fluentui/react-tabs`
`@fluentui/react-text`
`@fluentui/react-theme-provider`
`@fluentui/react-toggle`
`@fluentui/react-utilities`
`@fluentui/react-compose`

Deleted `next` folder from `@uifabric/react-cards`.

#### Focus areas to test

(optional)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "a11ytest": "cd apps && cd a11y-tests && yarn test",
     "build": "lage build --verbose",
     "build:fluentui:docs": "gulp build:docs",
-    "build:min": "yarn build --to @fluentui/react-next --to @fluentui/react-northstar --min",
+    "build:min": "yarn build --to @fluentui/react-northstar --min",
```
Comment (ecraig12345) [inline package.json:14]:
```suggestion
    "build:min": "yarn build --to @fluentui/react --to @fluentui/react-northstar --min",
```

File: packages/storybook/src/decorators/withKeytipLayer.tsx [hunk 2]
```diff
@@ -10,6 +10,6 @@ export const KeytipLayerWrapper: React.FunctionComponent<{}> = props => {
   );
 };
 
-export const withKeytipLayer = (storyFn: () => React.ReactNode) => {
-  return <KeytipLayerWrapper>{storyFn()}</KeytipLayerWrapper>;
+export const withCompatKeytipLayer = (storyFn: () => React.ReactNode) => {
```
Comment (ecraig12345) [inline packages/storybook/src/decorators/withKeytipLayer.tsx:13]: I think you forgot to change the name of this to withKeytipLayer? (or need to update references elsewhere)

File: scripts/lint-files/no-old-snapshot-paths.js [hunk 3]
```diff
@@ -9,10 +9,9 @@ const { findGitRoot } = require('../monorepo/index');
 function lintFiles() {
   const gitRoot = findGitRoot();
 
-  const exampleFiles = glob.sync(
-    'packages/{office-ui-fabric-react,react-cards,react-focus,react-next}/src/components/__snapshots__/*',
-    { cwd: gitRoot },
-  );
+  const exampleFiles = glob.sync('packages/{office-ui-fabric-react,react-focus/src/components/__snapshots__/*', {
```
Comment (ecraig12345) [inline scripts/lint-files/no-old-snapshot-paths.js:12]:
```suggestion
  const exampleFiles = glob.sync('packages/{office-ui-fabric-react,react-focus}/src/components/__snapshots__/*', {
```

**Comment by ecraig12345** [comment]
We might need to wait on this until immediately before we're ready to publish. And/or @dzearing should re-target #15588 against master (may apply to some other PRs too).

**Review by khmakoto — COMMENTED** [review]
We should probably ask @dzearing if it's ok to remove the `react-button` package or if it is needed in v7 for his work on the WAC Ribbon.

**Review by khmakoto — APPROVED** [review]

**Review by ecraig12345 — APPROVED** [review]

**Pull request merged**